pypeit/pypeit.py

```diff
@@ -683,6 +684,9 @@ def reduce_one(self, frames, det, bg_frames, std_outfile=None):
                                         bpmmask=outmask,
                                         detector=sciImg.detector,
                                         sci_spat_flexure=sciImg.spat_flexure,
```

Comment:
It would be simpler to me if we just passed in a dictionary or satrapy table with meta data, rather than a specific argument for each type of meta data, since that argument list could continue to grow.